src/pages/Content/index.js

```diff
@@ -1,15 +1,147 @@
+
+var recordedEvents = []; // store events
+var clipboard = []; // store user copied text
```

Comment:
Let's split this to another PR? it makes this one too big to review and it isn't part of the MVP.